<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QvtkDiscreteMarchingCubes</class>
 <widget class="QWidget" name="QvtkDiscreteMarchingCubes">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>38</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Start:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QSpinBox" name="spinBoxStart"/>
   </item>
   <item>
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>End:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QSpinBox" name="spinBoxEnd"/>
   </item>
   <item>
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>SmoothingIteration:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QSpinBox" name="spinBoxSmoothingIterations"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
